apps/docs/content/components/CTAForm.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: CTAForm
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/CTAForm/CTAForm.tsx
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=1377%3A30754'
+storybook: '/brand/storybook/?path=/story/components-CTAForm--playground'
+description: Use CTAForm to create a form that is used to collect minimal information from users.
+---
+
+import {
+  CTAForm,
+  InlineLink,
+  FormControl,
+  TextInput,
+  Checkbox,
+} from '@primer/react-brand'
+
+```js
+import {CTAForm} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<CTAForm>
+  <CTAForm.Input>
+    <FormControl required size="large" fullWidth>
+      <FormControl.Label>Your Work Email Address</FormControl.Label>
+      <TextInput placeholder="name" />
+    </FormControl>
+  </CTAForm.Input>
+  <CTAForm.Confirm>
+    <FormControl required size="large" fullWidth>
+      <FormControl.Label>
+        I agree to the <InlineLink href="#">Privacy Policy</InlineLink> and{' '}
+        <InlineLink href="#">Terms of Service</InlineLink>
+      </FormControl.Label>
+      <Checkbox name="confirm" />
+    </FormControl>
+  </CTAForm.Confirm>
+  <CTAForm.Action>Subscribe</CTAForm.Action>
+</CTAForm>
+```
+
+## Component props
+
+### CTAForm
+
+| name        | type     | default     | required | description                                                        |
+| ----------- | -------- | ----------- | -------- | ------------------------------------------------------------------ |
+| `action`    | `string` | `undefined` | `false`  | The URL that processes the form submission                         |
+| `method`    | `string` | `undefined` | `false`  | The HTTP method to submit the form with                            |
+| `target`    | `string` | `undefined` | `false`  | Indicates where to display the response after submitting the form. |
+| `className` | `string` | `undefined` | `false`  | Sets a custom class                                                |
+
+```jsx live
```

Comment:
Are these snippets supposed to be here? Can we hoist them up to go alongside the other live examples please? This one seems to be showing the component without confirmation child, so guessing it should have it's own header too?

Reply:
I think that was the intention I will make that update.